aya/src/obj/mod.rs

```diff
@@ -1939,6 +2209,85 @@ mod tests {
             .unwrap();
 
         let map = obj.maps.get(".rodata").unwrap();
-        assert_eq!(test_data, map.data);
+        assert_eq!(test_data, map.data());
+    }
+
+    #[test]
+    fn test_parse_btf_map_section() {
+        let mut obj = fake_obj();
+        fake_sym(&mut obj, 0, 0, "my_map", 0);
+        let data: &[u8] = &[
```

Comment:
Add comment on how you generated this pls? So when it inevitably starts failing in 2 years we can generate it again :P